.github/ISSUE_TEMPLATE/hotshot-general-task-template.md

```diff
@@ -0,0 +1,27 @@
+---
+name: HotShot General Task Template
+about: For general tasks in HotShot
+title: "[MACRO_TASK_SHORT_NAME] Short task description"
+labels: ''
+assignees: elliedavidson
+
+---
+
+### Describe the task
```

Comment:
We could also add these to the description:
* Task dependencies
* Solution trade-offs